packages/core/src/repository/mapper/mapper.ts

```diff
@@ -137,3 +137,13 @@ export class PaginationPageMapper<From, To> implements Mapper<PaginationPage<Fro
         );
     }
 }
+
+/**
+ * Maps an array of objects
+ */
+export class ArrayMapper<From, To> implements Mapper<From[], To[]> {
+    constructor(private readonly mapper: Mapper<From, To>) {}
+    map(from: From[]): To[] {
```

Comment:
Nit. Not sure if this is part of our style, but I tend to be explicit with the visibility:

```suggestion
    public map(from: From[]): To[] {
```

Reply:
It is. But the rest of the mappers look the same, I'll change this for all of them. Thanks!

Reply:
Will you do it in the context of this PR?

Reply:
As we talked into the strict-mode conversations, all these refactors to update harmony to our current standards will be managed in a future PR, so I finally changed the visibility only for this new mapper